### PR TITLE
Add `datacenter` label to kubermatic_cluster_info

### LINKED ISSUE #2245: add datacenter label to kubermatic_cluster_info metric
`kubermatic_cluster_info` metric contains lots of useful into about state of the clusters such as cloud provider, id, name, master version, etc. It's useful for "general overview" of installation.

Would be even more useful if `datacenter` label will be available, `count(kubermatic_cluster_info) by (cloud_provider, datacenter)` could be used to quickly see the state of the clusters by provider/datacenter

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
For `count(kubermatic_cluster_info) by (cloud_provider, datacenter)` (or similar) prometheus query.

Fixes #2245 

```release-note monitoring
Add datacenter label to kubermatic_cluster_info metric
```

### CODE REVIEW

**Comment by xrstf** [comment]
I wonder why your dev environment does not sort imports like mine does. There was a reason why all kubermatic imports were grouped together.

**Review by xrstf — DISMISSED** [review]
Apart from the weird re-ordering of imports this LGTM.

**Comment by kron4eg** [comment]
@xrstf that's not dev environment, that's me. I've resorted and regrouped imports in order "stdlib, 3rdparty libs, ourlibs". Basically the habit from [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).

File: api/pkg/collectors/cluster.go [hunk 1]
```diff
@@ -4,15 +4,14 @@ import (
 	"fmt"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/apimachinery/pkg/labels"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	kubeinformers "k8s.io/client-go/informers"
```
Comment (mrIncompetent) [inline api/pkg/collectors/cluster.go]: `k8s.io` imports get a dedicated block in our codebase

Comment (kron4eg) [reply api/pkg/collectors/cluster.go]: Good to know, where can I read our styling guide about this?

Comment (kron4eg) [reply api/pkg/collectors/cluster.go]: @mrIncompetent ping

Comment (mrIncompetent) [reply api/pkg/collectors/cluster.go]: Not documented. 
Its basically:
Whatever enforces our linter + grouping of imports.
Grouping should be done with:
- Single group for go sdk packages
- Single group for own project packages (Here: `github.com/kubermatic/kubermatic`) 
- Single project for `k8s.io` packages

I'll create a markdown document explaining that.

**Review by xrstf — APPROVED** [review]

**Review by mrIncompetent — APPROVED** [review]

**Pull request merged**